dbld/images/required-pip/all.txt

```diff
@@ -1,6 +1,6 @@
-nose==1.3.7
-ply==3.10
-pep8==1.7.1
-pylint==1.8.2
-astroid==1.6.1
-logilab-common<=0.63.0
+nose
+ply
+pycodestyle
```

Comment:
We should use `pycodestyle` in `pylib/Makefile.am` if we depend on the new name (this is why the macOS build fails).

https://github.com/balabit/syslog-ng/blob/28a34968d572ab6dfecf71d9d3fff1b6ccb9ecae/modules/python/pylib/Makefile.am#L71-L72

Reply:
ahh.. I've removed the rest of the branch, one of the patches did autodetected whether to use
pep8 or pycodestyle.

I think we can work with the warning that pep8 generates, so I change this patch to use pep8 instead.